easybuild/easyconfigs/j/Java/README.md

```diff
@@ -0,0 +1,4 @@
+In order to use these configs you must first download Java from [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/index.html).  
+
+To install, you must past eb the sourcepath to find the download. For example, if the file is in /tmp your command might look something like:
```

Comment:
You're right that EasyBuild only looks in particular places for 'sources', but the way it's stated now is not how it's typically done.

You usually have configured EasyBuild with a particular `sourcepath` location, either directly via the `--sourcepath` configuration option (via a configuration file or via `$EASYBUILD_SOURCEPATH`), or indirectly via `--prefix` (or `$EASYBUILD_PREFIX` or cfg file).

So typically you'll copy/move the downloaded tarball for Java into `<sourcepath>/j/Java`, since that's also where EasyBuild will download stuff into (when it can).